assets/js/wfmodule/WfModuleHeader.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import StatusBar from './StatusBar';
+
+export default class WfModuleHeader extends React.Component {
+  constructor(props) {
+    super(props);
+    this.setModuleRef = this.setModuleRef.bind(this);
+  }
+
+  componentDidMount() {
+    if (this.props.isSelected) {
+      this.props.focusModule(this.moduleRef);
+    }
+  }
+
+  setModuleRef(ref) {
+    this.moduleRef = ref;
+  }
+
+  shouldComponentUpdate(nextProps) {
+    return nextProps.moduleName !== this.props.moduleName;
```

Comment:
I see that render() uses props.isSelected. So if isSelected changes, we won't see it until the module name changes. Is this correct?